tests/rest/datacube/test_datacube.py

```diff
@@ -411,7 +412,7 @@ def check_request(request):
     )
 
     res = s2cube.tiled_viewing_service(type="WMTS", title="S2 Foo", description="Nice!", custom_param=45)
-    assert res == {"url": API_URL + "/services/sf00", 'service_id': 'sf00'}
+    assert res.service_id == Service('sf00', connection).service_id
```

Comment:
`Service('sf00', connection).service_id` ?
why not just `'sf00'`?

Reply:
No clue, can't remember.

```suggestion
    assert res.service_id == 'sf00'
```